src/main/java/pl/futurecollars/invoice/utils/FilesService.java

```diff
@@ -6,13 +6,16 @@
 import java.nio.file.Path;
 import java.nio.file.StandardOpenOption;
 import java.util.List;
+import org.springframework.stereotype.Component;
 
+@Component
 public class FilesService {
```

Comment:
@Service

Reply:
Jeśli FileService oraz Json Service są używane tylko, gdy mamy do czynienia z plikową bazą danych, to deklaracje tych beanów również warto przenieść do FileDatabaseConfiguration.